Taggers/python/tagsDumpConfig_cff.py

```diff
@@ -6,7 +6,8 @@
     generatorInfo = cms.InputTag("generator"),
     processId = cms.string(""),
     maxCandPerEvent = cms.int32(-1),
-    lumiWeight = cms.double(1.0),
+    lumiWeight = cms.double(1.0),# Over-written by metaData if using fggRunJobs
+    intLumi = cms.untracked.double(1000.), # in /pb. to be over-written by metaData if using fggRunJobs
```

Comment:
Pleas add this also to the other dumpers, or else make the call on line 189 conditional on having the parameter (and take line 122).

`if( cfg.exists("IntLumi") ) { 
      RooRealVar* intLumi  = ...
}`